### DISCUSSION TITLE
[Accordion] Does prop `padding` make sense?

### DISCUSSION THREAD

**Discussion by GomezIvann** [dxc-technology/halstack-react#698] [Ideas] [answered]
My approach is the same as for tokens in this situation, should we have props that modify or affect the custom content of container components, such as this one or Card, or Dialog, etc? I don't think this prop give too much value and opens the door to ask why not other properties such as margins, alignment, etc. 

This can also be applied to Card's prop `contentPadding`.

**Comment by aweell** [comment]
I don't think we're adding so much value with this `prop` other than providing a set of pre-defined values since it's as easy as using a container with custom styles wrapping the content:

![Screen Shot 2022-02-16 at 1 41 12 PM](https://user-images.githubusercontent.com/44420072/154266791-36cad4f3-618e-4e55-b722-cf40bd7f9288.png)

I would understand the case when we allow changing the `padding` of the elements we provide by default (e.g. change the padding of the accordion heading container, which affects the label, helper text, and icon)

Frankly speaking, I wouldn't miss it if it's not there since we're developing other ways to apply `padding`, and component spacing in our layout proposal...

**Reply by GomezIvann** [reply]
I totally agree with you Alex, I don't think we are giving too much value with this prop, since it can be done in a few lines and is much more customizable.

**Comment by GomezIvann** [comment, marked as answer]
Since the release of the Layout components, we have decided [here](https://github.com/dxc-technology/halstack-react/discussions/1266) that removing `padding` props is the first step of the new strategy about how we want users to create layouts and arrange items on the screen.
